pycontrails/physics/jet.py

```diff
@@ -24,14 +24,62 @@
 # -------------------
 
 
+def acceleration(
+    true_airspeed: npt.NDArray[np.float_], dt: npt.NDArray[np.float_]
```

Comment:
We call this `segment_duration` in `jet.fuel_burn`. I think I prefer the verbose name.

```
    segment_duration: npt.NDArray[np.float_]
        Time difference between waypoints, [:math:`s`]
```

We should add a `See Also` to both of these for `flight.segment_duration`
```
    See Also
    --------
    :func:`flight.segment_duration`

```

Reply:
9a57e5e